markdown-link-check.config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "retryOn429": true,
+  "httpHeaders": [
+    {
+      "urls": ["https://ngrx.io/"],
+      "headers": {
+        "Accept-Encoding": "zstd, br, gzip, deflate"
+      }
+    }
+  ],
+  "retryCount": 2,
+  "fallbackRetryDelay": "1s",
+  "ignorePatterns": [
+    {
+      "pattern": "^https://ngrx.io/"
```

Comment:
hint for reviewers: 
Unfortunately the link-check fails for every ngrx.io-url even if they are valid
I do not really know why this is the case so far.
Also created an issue to investigate this further in `markdown-link-check`, see https://github.com/tcort/markdown-link-check/issues/229